cli/flox-activations/src/cli/activate/mod.rs

```diff
@@ -392,6 +393,20 @@ impl ActivateArgs {
 
         match context.shell {
             ShellWithPath::Bash(_) => {
+                // TODO: I think this is wrong.
```

Comment:
This comment is unclear. Are you expecting not to see output when `flox activate` is piped to `cat`? Is that output from `main` or from the current implementation?

Reply:
Tried to clarify